core/controllers/story_editor.py

```diff
@@ -59,6 +60,49 @@ class EditableStoryDataHandler(base.BaseHandler):
     """A data handler for stories which support writing."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'story_id': {
+            'schema': {
+                'type': 'basestring'
+            },
+            'validators': [{
+                'id': 'has_length',
+                'value': constants.STORY_ID_LENGTH
+            }]
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'PUT': {
+            'version': {
+                'schema': {
+                    'type': 'int',
+                    'default_value': None
```

Comment:
Why can this be `None`?

Reply:
The `_require_valid_version` validator in `story_editor.py` checks if the version is invalid and I figured it was better to explicitly define that outcome within the schema. Should I remove it?

Reply:
Not sure what you mean.